src/units/unit.cpp

```diff
@@ -1496,7 +1514,7 @@ void unit::write(config& cfg, bool write_all) const
 	cfg.clear_children("events");
 	cfg.append(events_);
 
-	cfg["overlays"] = utils::join(overlays_);
+	cfg["overlays"] = utils::join(meta_overlays_);
```

Comment:
It seems as if writing a unit to a config and then re-reading it from that config will move `meta_overlays_` to `overlays_`. I don't understand the differences enough to know if that's correct or a bug.

Reply:
here, cfg["overlays"] is the wml variable used by example for conditonnal events. by example
[if]
[variable]
name=unit.overlays
contains=image
if unit.overlays contains overlay_ data only then obj_overlays_ data are ignored, and all data overlays must be used here. overlays_ must be replaced by meta_overlays_ in 1517 line.